Source/Session.swift

```diff
@@ -34,6 +34,7 @@ open class Session {
     public let adapter: RequestAdapter?
     public let retrier: RequestRetrier?
     public let serverTrustManager: ServerTrustManager?
+    public let redirectHandler: RedirectHandler?
```

Comment:
Similar to the `Request` side, do we want the optional property or an array?

Reply:
Based on Slack conversation, we've decided that this would overcomplicate this feature with little upside and would most likely lead to confusion on the client side.